cache/redis/cache.go

```diff
@@ -24,8 +24,9 @@ func (c *cache) Get(key string) (resp []byte, ok bool) {
 }
 
 // Set saves a response to the cache as key.
-func (c *cache) Set(key string, resp []byte) {
-	c.conn().Do("SET", cacheKey(key), resp)
+func (c *cache) Set(key string, value []byte) {
+	_5Days := 5 * 60 * 60 * 24
```

Comment:
:information_source: I think a better name for the variable would be `ttl` or `exp`. Then you could document with a comment, like the following:
```go
exp := 5 * 60 * 60 * 24 // five days
```